.github/workflows/ci-tests.yml

```diff
@@ -71,7 +71,7 @@ jobs:
       - name: Run Gateway Tests
         id: ci-tests
         run: |
-          ./bin/ci-tests.sh 2>&1 | tee test.log
+          ./ci/bin/ci-tests.sh 2>&1 | tee test.log
```

Comment:
This file will need to be left out for now. This file is owned by the squads.

Reply:
Then will keep a `bin/` directory in the root, with this file there, and the rest will be kept in `ci/bin`